### ISSUE TITLE
App is running in background on Windows

### ISSUE THREAD

**Issue by quanglam2807** [quanglam2807/appifier#206]
```
Jon Nelson

SOMETIMES an app will be running (such as Gmail or Google contacts or Google Calendar) and after minimizing the app I won't be able to open it again. My desktop shows me that the app is already open (the icon has a tick on it when an app is open) but I can't find the app open when using the open windows viewer (see screenshots, where Gmail is NOT visible but seems to be running in background)  When I click on the desktop icon it seems to "try to open" for a few seconds....but nothing happens.  

I don't know how to reproduce the issue, it seems to happen randomly.

Thanks for you help
```

![gmail running in background](https://user-images.githubusercontent.com/1548835/33124665-5b567ec0-cfb0-11e7-97dd-ea16dfa2138a.PNG)
![gmail not showing](https://user-images.githubusercontent.com/1548835/33124670-5d013cec-cfb0-11e7-8005-00a2a414709a.png)

**Comment by fsdias** [comment]
I have observed a similar situation on Ubuntu 16.04. In this case Facebook was opened and closed a few times and at some point I was not able to open it again. It was only after I manually ended the multiple Facebook processes that I was able to run it again. 

Running Webcatalog from the command line and running Facebook from within it does not throw any error

![screenshot from 2017-12-13 11-34-53](https://user-images.githubusercontent.com/5689181/33937026-ac6049aa-dff9-11e7-8799-ffda1aacc531.png)
